lib/phone_shop/screens/home_screen.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/phone_shop/widgets/components/fill_button.dart';
+import 'package:flutter_internship/phone_shop/widgets/components/shop_card.dart';
+import 'package:flutter_internship/phone_shop/widgets/data/shop_cards.dart';
+import 'package:flutter_svg/svg.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFFF8F8F8),
+      appBar: _HomeScreenTitle(),
+      body: SafeArea(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            _ShopCardList(),
+            _DisabledButton(),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _HomeScreenTitle extends StatelessWidget implements PreferredSizeWidget {
+  @override
+  Size get preferredSize => const Size.fromHeight(kToolbarHeight);
+
+  @override
+  Widget build(BuildContext context) {
+    return PreferredSize(
+      preferredSize: const Size.fromHeight(102),
```

Comment:
If you implement PreferredSizeWidget there is no need to wrap everything in PreferredSize.